contracts/Rewards.sol

```diff
@@ -49,10 +49,10 @@ contract Rewards is IRewards, ERC20AccessorWithTokenGranularity, ManagedContract
         address _registryManager,
         IERC20 _erc20,
         IERC20 _bootstrapToken,
-        uint48 generalCommitteeAnnualBootstrap,
-        uint48 certifiedCommitteeAnnualBootstrap,
-        uint48 annualRateInPercentMille,
-        uint48 annualCap,
+        uint generalCommitteeAnnualBootstrap,
+        uint certifiedCommitteeAnnualBootstrap,
+        uint annualRateInPercentMille,
```

Comment:
better to use uint32 similarly to maxDelegatorsStakingRewardsPercentMille to avoid confusion